File: 프로그래머스/0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.js
function solution(s) {
  let result = [];
  const object = s.split('').reduce((accu, curr) => { 
    accu[curr] = (accu[curr] || 0)+1; 
    return accu;
  }, {});
  
  for (const key in object) {
    object[key] === 1 ? result.push(key) : result
  }
  return result.sort().join("")
}

File: 프로그래머스/0/120896. 한 번만 등장한 문자/한 번만 등장한 문자.java
import java.util.Arrays;

class Solution {
        public static String solution(String s) {
        int[] count = new int[26];
            
        for (char c : s.toCharArray()) {
            count[c - 'a']++;
        }

        StringBuilder answer = new StringBuilder();
        for (char c : s.toCharArray()) {
            if (count[c - 'a'] == 1) {
                answer.append(c);
            }
        }
        char[] result = answer.toString().toCharArray();
        Arrays.sort(result);

        return new String(result);
    }
}